### PR TITLE
FIX : #4768

### PR DESCRIPTION
**Pull request by FHenry**
FIX : #4768

### CODE REVIEW

File: htdocs/install/mysql/migration/3.7.0-3.8.0.sql [hunk 1]
```diff
@@ -797,3 +797,14 @@ ALTER TABLE llx_societe_remise_except MODIFY COLUMN description text NOT NULL;
 update llx_opensurvey_sondage set format = 'D' where format = 'D+';
 update llx_opensurvey_sondage set format = 'A' where format = 'A+';
 
+--Deal with holidays_user that do not have rowid
```
Comment (eldy) [inline htdocs/install/mysql/migration/3.7.0-3.8.0.sql:800]: I reverted that. No database structure change is allowed into a minor version.
Why this change ? A rowid is not required here, the key is couple fk_user/fk_type and code should rely on this.

**Comment by FHenry** [comment]
@eldy Laurent, If you do not define rowid Mysql will automaticly set fk_user as Primary key on table creation because it is the first integer. The only option is to fix this bug is add rowid as primary key. This is not this first time we need to update Db model to fix stuff...
Do the test, you will see there is no alternative.
